js/popup.js

```diff
@@ -55,23 +61,57 @@ const createComment = (comment) => {
   return commentFragment;
 };
 
+const updateCommentCount = (shownComments, allComments) => {
+  commentCount.innerHTML = `${shownComments} из <span class="comments-count">${allComments}</span> комментариев`;
+};
+
+const updateCommentLoader = (shownComments, allComments) => {
+  if (shownComments === allComments.length) {
+    commentLoader.classList.add('hidden');
+  }
+};
+
+
 const addBigPictureComments = (comments) => {
   socialComments.innerHTML = '';
   const fragment = document.createDocumentFragment();
 
-  comments.forEach((comment) => {
+  comments.forEach((comment, index) => {
     const commentFragment = createComment(comment);
     fragment.appendChild(commentFragment);
+    if (index >= initialComments) {
+      fragment.children[index].classList.add('hidden');
+    }
   });
 
   socialComments.appendChild(fragment);
+
 };
 
+const showComments = function () {
+  const li = socialComments.querySelectorAll('li');
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice

- достаем все комментарии при помощи `socialComments.children`
- "вырезаем нужные для отображения комментарии
- удаляем у них класс `hidden`

```suggestion
  Array
    .from(socialComments.children)
    .slice(commentsCounter, commentsCounter + COMMENTS_STEP)
    .forEach((comment) => {
      comment.classList.remove('hidden');
    });

  commentsCounter = commentsCounter + COMMENTS_STEP;
  
  updateCommentCount(...);
  updateCommentLoader(...);
```